index.html

```diff
@@ -59,227 +60,218 @@ <h1>I'm Abdi</h1>
         <li><a href="#" class ="link"><img src="./assets/images/M.svg"></a></li>
      </ul>
  </div>
-    
+
+
 </section>
     </div>
+       
+            <!-- ******* Work Section  *******-->  
     <section class="container">
         <div class="works-Section">
             <h2 class="recentWorks_header">My Recent Works</h2>
+            <img src="./assets/images/besideWorkLine.png" id="besiderecentWorks">
+            
+            <!--Card1-->
             <div class="cards card1">
```

Comment:
The section containing the cards should use a a `display:grid` to arrange the cards. This is a requirement for this milestone that both in your mobile design and desktop you use `display:grid`  for the cards and `display:flex` to arrange the card contents
<img width="1680" alt="Screenshot 2022-09-30 at 04 41 47" src="https://user-images.githubusercontent.com/48546992/193352813-70a3356b-01bd-4b6a-8156-67dcdc30ddad.png">